### ISSUE TITLE
[Feature] Quickly create pre-defined markers

### ISSUE THREAD

**Issue by darwin1839** [stashapp/stash#1052]
**Is your feature request related to a problem? Please describe.**
I create the same handful of markers (same title, default tag, tags) on almost every scene. It would be nice to have the ability to quickly create these.

**Describe the solution you'd like**
A way to define a few custom marker shortcuts/keyboard shortcuts that would open the new marker form with defined title, default tag, and tags entered already

**Describe alternatives you've considered**
Creating markers manually

**Additional context**
Buttons in the UI  or Keyboard shortcuts (number keys?) would be fine

**Comment by jeremymeyers** [comment]
Do you mean markers specifically? if you mean tags, you can bulk edit scenes with the Tagger interface

**Comment by darwin1839** [comment]
Yeah, markers. When watching a scene, it would be nice to just press a hotkey and have a marker created without having to pause and fill out the form.

**Comment by jeremymeyers** [comment]
oh, yes, a hotkey for 'create market at this point' would be nice. We might run into trouble depending on whether the player itself has focus or not
